index.html

```diff
@@ -1,15 +1,59 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <title>Document</title>
-    <link rel="stylesheet" href="style.css">
-    <link href="https://fonts.googleapis.com/css?family=Barlow+Semi+Condensed" rel="stylesheet">
-  </head>
-  <body>
-    <main id='storeName'>
-      <ul id='parent'>
-        <il id='child'></il>
-      </ul>
-    </main>
-    <script src="app.js"></script>
-  </body>
+<head>
+  <title>Cookie Stand</title>
+  <link rel="stylesheet" href="style.css">
+  <link href="https://fonts.googleapis.com/css?family=Barlow+Semi+Condensed" rel="stylesheet">
+</head>
+
+<body>
+  <<header class="mainheader">
```

Comment:
you need to be naming classes and id's in kabob-case or snake_case, separating words if there are more than one